spec/controllers/surveys_controller_spec.rb

```diff
@@ -1,5 +1,76 @@
 require 'rails_helper'
 
 describe SurveysController, type: :controller do
+  context 'send survey validation' do
 
+    it 'respondents is empty' do
+      FactoryGirl.create(:survey, :id => 1)
+      post :send_survey, :id => 1, :respondent_phone_numbers => '[]'
+
+      expect(response.status).to eq(400)
+      json = JSON.parse(response.body)
+      expect(json['error']).to eq('respondents_is_empty')
+    end
+
+    it 'respondents is created' do
+      FactoryGirl.create(:survey, :id => 1)
+      expect {
+        post :send_survey, :id => 1, :respondent_phone_numbers => '["1-647-111-1111"]'
+      }.to change { Respondent.count }.by(1)
+    end
+
+    it 'respondents is not created' do
+      FactoryGirl.create(:survey, :id => 1)
+      FactoryGirl.create(:respondent, :phone_number => '1-647-111-1111')
+      expect {
+        post :send_survey, :id => 1, :respondent_phone_numbers => '["16471111111"]'
+      }.to_not change { Respondent.count }
+    end
+
+    it 'survey doesn\'t exist' do
+      post :send_survey, :id => 1, :respondent_phone_numbers => '["1-647-111-1111"]'
+
+      expect(response.status).to eq(404)
+      json = JSON.parse(response.body)
+      expect(json['error']).to eq('survey_not_found')
+    end
+
+    it 'survey first question doesn\'t exist' do
```

Comment:
Personally prefer for it statements to describe the behaviour you expect like 'throws _x_ error' or 'creates a respondent'. There's something called `context` which is typically used to group several it statements under a certain set of conditions like:
```
context "survey first question doesn't exist" do
  it "throws survey_not_sendable" do
    # test
  end
end
```
That being said there would be one spec under each context so there's not a lot of value in nesting. Maybe these could read `it "throws error when first question doesn't exist"` or something like that?